config/host.exs

```diff
@@ -3,10 +3,12 @@ use Mix.Config
 config :vintage_net_wizard,
   backend: VintageNetWizard.Backend.Mock
 
-  # We don't run a server during test. If one is required,
+# We don't run a server during test. If one is required,
 # you can enable the server option below.
 config :drizzle_ui, DrizzleUiWeb.Endpoint,
   http: [port: 4002],
   server: true
 
-# config :drizzle, gpio_module: Drizzle.GPIO
+config :drizzle,
+  # gpio_module: Drizzle.GPIO
+  schedule_dir: Path.expand(Path.join(__DIR__, "../tmp"))
```

Comment:
why temporary?

Reply:
This would be `drizzle/tmp` it will exist as long as you want it to. Mostly out of habit, but that's where I put things locally when developing that I don't need in the repo and consider transient.

Prob should add this to the .gitignore...